hack/lib/common-var-init.sh

```diff
@@ -161,6 +162,7 @@ API_SECURE_PORT=${API_SECURE_PORT:-6443}
 # WARNING: For DNS to work on most setups you should export API_HOST as the docker0 ip address,
 API_HOST=${API_HOST:-"${lohostname}"}
 API_HOST_IP=${API_HOST_IP:-"0.0.0.0"}
+API_HOST_IP_EXTERNAL=${API_HOST_IP_EXTERNAL:-"${hostip}"}
```

Comment:
nit: seems we can move the definition of hostip from line 81 into line 165 because the variable hostip is only used here.  same to how line 163 works.

Reply:
also, consider adding some info explaining API_HOST_IP_EXTERNAL now that we have API_HOST, API_HOST_IP and API_HOST_IP_EXTERNAL. It can get a bit confusing.

In fact please add this to the PR description why this variable is introduced in the cert.

Reply:
Kept where lohostname is defined